src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/ace/ace-uncompressed.js

```diff
@@ -9696,81 +9696,83 @@ oop.inherits(Mode, CMode);
 exports.Mode = Mode;
```

Comment:
@jjallaire @kevinushey is it generated by rstudio? How should I regenerate this? If not, should the rules I've added take precedence over this file? Currently there are not, so I'll take a closer look.